packages/components/src/slot-fill/bubbles-virtually/fill.tsx

```diff
@@ -20,18 +19,15 @@ import type { FillComponentProps } from '../types';
 export default function Fill( { name, children }: FillComponentProps ) {
 	const registry = useContext( SlotFillContext );
 	const slot = useObservableValue( registry.slots, name );
-	const [ , rerender ] = useReducer( () => [], [] );
-	const ref = useRef( { rerender } );
+	const instanceRef = useRef( {} );
```

Comment:
The `rerender` API goes away and the fill now registers only an unique instance object. I'm renaming it from `ref` to `instance` when registering, something that @tyxla asked for in one of the previous PRs.